OJS.Workers.Executors/StandardProcessExecutor.cs

```diff
@@ -94,6 +92,17 @@ public StandardProcessExecutor(int baseTimeUsed, int baseMemoryUsed, ITasksServi
                     timeSamplingThreadInfo = this.StartProcessorTimeSamplingThread(process, result);
                 }
 
+                try
+                {
+                    result.ReceivedOutput = processOutputTask.GetAwaiter().GetResult();
+                    result.ErrorOutput = errorOutputTask.GetAwaiter().GetResult();
+                }
+                catch (Exception ex)
+                {
+                    logger.Error("Error while reading the process streams.", ex);
+                    result.ErrorOutput = $"Error while reading the process streams. {ex.Message}";
```

Comment:
Is this `ErrorOutput` forwarded to the end user? We should make it only visible for admins.